src/GafferUI/StandardGraphLayout.cpp

```diff
@@ -484,6 +507,173 @@ class LayoutEngine
 			}
 		}
 
+		V3f getDefaultTangent( const NodeGadget *dstNodeGadget )
```

Comment:
Let's make this private, to make the relationship between LayoutEngine and StandardGraphLayout clearer. Let's also call it `defaultTangent()` to avoid any expectation of a matching `setDefaultTangent()`.